python/dtpcontrols/setup.py

```diff
@@ -1,13 +1,31 @@
 # eveyrthing to do with ipbus connection to the device goes here...
 
 import os
+import string
 import uhal
 from ctypes import cdll
 
-def connectionManager(conn_file=os.environ['CONNECTION_FILE']):
+def find_conn_file():
+    path = os.environ.get('DUNEDAQ_SHARE_PATH')
```

Comment:
Is it `DUNEDAQ_SHARE_PATH`, or `DTPCONTROLS_SHARE_PATH`? Using the latter should reduce the already low likelihood of conflicts...

Reply:
Ah!  It is deliberately `DUNEDAQ_SHARE_PATH` because I could see no evidence of anything like this that was package-related.  But if `DTPCONTROLS_SHARE_PATH` is an acceptable option I will change.  How/where should this be set though?

Reply:
It's supposed to be automagically set by the `dbt-work-env`.

Reply:
Confirmed, with a minor correction. For every package, either in ups or source code form, `dbt-work-env` makes available a `<PACKAGE>_SHARE` environment variable that points to the package share area (where the connection file is installed)

Reply:
awesome, thanks.  fixed.